### PR TITLE
feat(secu): update security policy to redirect to VDP

### PR DESCRIPTION
**Pull request by sc979**
## Description

Update security policy to redirect to VDP

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -65,35 +71,11 @@ Centreon reserves the right to make final decisions regarding publishing acknowl
...
-* Proof-of-concept or exploit code (if possible)
-* Impact of the issue, including how an attacker might exploit the issue
-
-## Bug bounty
 
-We don't have a bug bounty program but this is something we are thinking about.
+# PGP information (Deprecated)
 
-## PGP information
+Kindly do not send us a mail or report a flaw using the Github issues.
```
Comment (vuntz) [inline SECURITY.md:77]: Github -> GitHub

File: SECURITY.md [hunk 2]
```diff
@@ -65,35 +71,11 @@ Centreon reserves the right to make final decisions regarding publishing acknowl
...
+Instead, use our [Vulnerability Disclosure Program](https://vdp.centreon.com) form
```
Comment (vuntz) [inline SECURITY.md:78]: Missing . at the end

File: SECURITY.md (hunk 1 above)
Comment (cg-tw) [inline SECURITY.md:77]:
```suggestion
Kindly do not send us an email or report a flaw using the Github issues.
```

File: SECURITY.md (hunk 2 above)
Comment (cg-tw) [inline SECURITY.md:78]:
```suggestion
Instead, use our [Vulnerability Disclosure Program](https://vdp.centreon.com) form.
```

File: SECURITY.md (hunk 1 above)
Comment (sc979) [inline SECURITY.md:77]:
```suggestion
Kindly do not send us a mail or report a flaw using the GitHub issues.
```

File: SECURITY.md [hunk 3]
```diff
@@ -65,35 +71,11 @@ Centreon reserves the right to make final decisions regarding publishing acknowl
...
-* Proof-of-concept or exploit code (if possible)
-* Impact of the issue, including how an attacker might exploit the issue
-
-## Bug bounty
 
-We don't have a bug bounty program but this is something we are thinking about.
+# PGP information (Deprecated)
 
-## PGP information
+Kindly do not send us a mail or report a flaw using the GitHub issues.
```
Comment (sc979) [inline SECURITY.md:77]:
```suggestion
Kindly do not send us an email or report a flaw using GitHub issues.
```

**Pull request merged**